Todo_CLI/todoCLI.js

```diff
@@ -0,0 +1,73 @@
+const readline = require('readline');
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout,
+});
+let list = []
+let incomplete = '[ ] '
+let complete = '[✓] '
+
+function start(){
+rl.question('\n(v) View • (n) New • (cX) Complete • (dX) Delete • (q) Quit\n', (menuItem) => {
+    if (menuItem === "v"){
+        view()
+    }
+    else if (menuItem === "n"){
+        add()
+    }
+    else if (menuItem[0] === "c"){
+        let x = parseInt(menuItem[1])
+        comp(x)
```

Comment:
Crashes if the user attempts to complete a non-existent task, eg. `c9` when only 1 task has been created